data/80-udisks2.rules

```diff
@@ -119,6 +119,7 @@ SUBSYSTEMS=="usb", ENV{ID_VENDOR_ID}=="05e3", ENV{ID_MODEL_ID}=="0727", ENV{ID_D
 
 # Apple Bootstrap partitions
 ENV{ID_PART_ENTRY_SCHEME}=="mac", ENV{ID_PART_ENTRY_TYPE}=="Apple_Bootstrap", ENV{UDISKS_IGNORE}="1"
+ENV{ID_FS_LABEL}=="Apple", ENV{ID_PART_ENTRY_TYPE}=="48465300-0000-11aa-aa11-00306543ecac|0xaf", ENV{UDISKS_IGNORE}="1"
```

Comment:
OK, I understand.  "ANACONDA" was here originally because the `macboot.img` was `dd`'ed to the partition.  That proved unreliable to `fsck.hfsplus`.  So now, `macboot.img` is mounted and copied to the HFS+ formatted partition.

So, lets label the filesystem `macboot`.   That should be both descriptive and avoidable.
I've force pushed a revision.

Reply:
Well, upon further consideration, and review of http://www.win.tue.nl/~aeb/partitions/partition_types-1.html, I used `sfdisk --part-type $device $partition 426F6F74-0000-11AA-AA11-00306543ECAC` for GPT, or
`sfdisk --part-type $device $partition 0xab` for MBR disks.
This change requires only the inclusion of `0xab` in the special DOS partition types set.